### ISSUE TITLE
Profil Auto

### ISSUE THREAD

**Issue by lolo19430** [asciidisco/plugin.video.netflix#648]
Hello everyone,
Can we log in automatically to a profile?

**Comment by jakermx** [comment]
yes, on settings enable Auto Login, then browse profiles and on the profile you want, open the context menu then select use sor autologin, and that´s it.
